client/styles/debug.css

```diff
@@ -49,7 +61,6 @@ a:active {
 .info {
     position: relative;
     z-index: 9999;
-    padding: 20px;
```

Comment:
I think we do need this padding - `debug.css` is also used for PHP errors, and without this padding the error message and stack trace touch the edges of the browser window, which looks a little odd:

<img width="850" alt="screen shot 2017-07-17 at 09 24 18" src="https://user-images.githubusercontent.com/1655548/28260183-c40f7622-6ad1-11e7-9f2b-d087f0edeed0.png">